santa/src/santa.cpp

```diff
@@ -74,14 +80,18 @@ void scrapeStream(std::istream& incoming,
                   bool save_to_archive = false) {
   std::string line;
   while (std::getline(incoming, line)) {
-    // explicitly filter to only include DENY events
-    if (line.find("decision=DENY") == std::string::npos) {
+    // explicitly filter to only include EXEC actions
+    if (line.find("action=EXEC") == std::string::npos) {
```

Comment:
This is going to dramatically increase the size of the table, and I can't tell from here if the increase in data is worthwhile. Do you have metrics collected for what the increased size is like?